src/main/java/uk/co/ramp/covid/simulation/Model.java

```diff
@@ -144,6 +153,16 @@ public boolean isValid() {
 
         RNG.seed(rngSeed + simulationID);
 
+        if (!outputDisabled) {
+            createOutputDirectory();
+            configureLoggerRedirects();
+        }
+
```

Comment:
The logging about possibly generating an RNG seed is happening before the logger is redirected, so please could you log the value we're passing to RNG.seed() after redirecting the logger?  (RNG.seed() is called just above here on line 154.)

Reply:
I've pushed this to below the redirect completely. Feels like the Model should handle the seed generation internally and not in main anyway.

Reply:
I'm afraid this won't work as it is, because isValid() requires the seed to be set.
(Perhaps the fact that you didn't spot this suggests we could do with a better test?)

I guess moving the call to isValid() could fix this, or maybe better to change isValid()?

Reply:
Just to clarify, as this looks confusing when I read back my comment:
The problem that I think I can see is that `optionallyGenerateRNGSeed()` is being called after `isValid()`, and they are both checking if the seed has been set.